Main.java

```diff
@@ -9,7 +10,25 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // adding scanner for user input
+    Scanner input = new Scanner(System.in);
+    
+    // asks user for input
+    System.out.println("Please enter an integer:");
+    
+    //lets user input number
+    int integer = input.nextInt();
+
+    //asks user for second number 
+    System.out.println("Please enter another integer:");
+   
+    //lets user input second number 
+    int number = input.nextInt();
+  
+    if( integer > number){
+    System.out.println( "Your numbers in ascending order are " + integer +"," +number);}
+    
+    else if (integer > number);{
+    System.out.println( "Your numbers in ascending order are " + number +"," +integer);}

```

Comment:
Watch your indentation. If there is code inside of an if statement, it should be TABBED over an extra bit to show it is inside of the if-statement. You also have a semi colon at the end of your if statement on line 31. This is a big nono. Finally, the closing bracket for your if statements should be on their own line.
```

if( integer > number){
    System.out.println( "Your numbers in ascending order are " + integer +"," +number);
}else if (integer > number){
    System.out.println( "Your numbers in ascending order are " + number +"," +integer);
}
```